deployments/anvilbox/environment.py

```diff
@@ -32,7 +32,9 @@ def mksrc(google_project,
           subgraphs,
           flags: int = 0,
           /,
-          prefix: Optional[str] = None):
+          prefix: Optional[str] = None) -> tuple[str, str]:
+    env, project = snapshot.split('_', 1)
+    project = '_'.join(project.split('_')[:-3])
```

Comment:
This doesn't really reflect the intent of my suggestion. My main issue with this section is how the `project` variable is initialized with parts of the string that aren't part of the project name. If you feel it's important to use unpacking here, then I suggest an alternative compromise between your initial approach and the one I suggested:

```suggestion
    env, *project, _, _, _ = snapshot.split('_')
    project = '_'.join(project)
```

Reply:
Also applies to `anvildev` and `anvilprod`

Reply:
I implemented your previous suggestion, the intend is clearer like you said.